File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        
        vector<int>b;
        unordered_map<int,int>v;
        for(int i:nums)
        {
            v[i]++;

        }
           vector<pair<int,int>>a(v.begin(),v.end());
           sort(a.begin(),a.end(),[](pair<int,int> &a ,pair<int,int> &b){
            return a.second>b.second;
           });
           int count=0;
        for(auto &i :a)
        {
           if(count<k)
           
                b.push_back(i.first);
            count++;

        }
     return b;   
    }
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        
        int n= nums.length;
        int[]a = new int[n];
        
        for(int i=0;i<n;i++)
        {
            int p=1;
            for(int j=i+1;j<n;j++)
            {
                p*=nums[j];
            }
             for(int k=i-1;k>=0;k--)
            {
                p*=nums[k];
            }
            a[i]= p;
        }
        return a;
    }
}  
